garage/tf/core/parameter.py

```diff
@@ -0,0 +1,42 @@
+"""Parameter function with tensorflow as the only dependency."""
+
+import tensorflow as tf
+from tensorflow.python.ops.gen_array_ops import broadcast_to
+
+
+def parameter(input_var,
+              length,
+              initializer=tf.zeros_initializer(),
+              dtype=tf.float32,
+              trainable=True,
+              name="parameter"):
+    """
+    Paramter function.
```

Comment:
It's not a function. It's either a network or a layer.

Reply:
Perhaps explain why we would want such a thing

Reply:
It is from @zhanpenghe

Reply:
You are responsible for every line of code you commit to this repository, regardless of its origin.

Reply:
Typo ("Paramter")